doc/source/management/data_at_rest_encryption.rst

```diff
@@ -80,6 +80,55 @@ Trying to add unencrypted table to this table space will result in an error:
   |Percona XtraBackup| currently doesn't support backup of encrypted general
   tablespaces.
 
+Binary log encryption
+=====================
+
+A new option, implemented since |Percona Server| :rn:`5.7.20-19`, is
+encryption of binary and relay logs, triggered by the
+:variable:`encrypt_binlog` variable.
+
+Besides turning :variable:`encrypt_binlog` ``ON``, this feature requires both
+`master_verify_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_master_verify_checksum>`_
+and `binlog_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_checksum>`_
+variables to be turned ``ON``.
+
+Distinct keys are used for binary and relay logs, and key identifiers include
+server UUIDs, which makes multi-source replication possible.
```

Comment:
This is not true or not yet true - depends on what we decide. The same key is used for encrypting both binlog and relay logs - this is how MariaDB implementation works and I do not see gain in having separate keys for relay log and binlog log. The key name does not contain UUID as each server has to use separate keyring storage (file for keyring_file and secret_mount_point for keyring_vault). The documentation needs updating in those regards.